content/en/blog/honey-data-collection/index.md

```diff
@@ -129,7 +129,7 @@ To demonstrate the scope of the profiles that Honey could create from this data,
 
-**Honey can infer all this information directly from the data they collected.** And the examples we gave only represent a tiny fraction of the information that Honey has. The 27 lines that the examples are based on, only make up just over 1&nbsp;% of the entries Honey has collected on Benni. We have focussed on non-product related page views. But in addition, Honey also knows every product that Benni looked at while he had installed the extension. Further, they could use the data they have for even more conclusions and profiles. For example, they could infer sleep cycles from the timestamps or build interest profiles based on the sites that were visited.
+**Honey can infer all this information directly from the data they collected.** And the examples we gave only represent a tiny fraction of the information that Honey has. The 27 lines that the examples are based on, only make up just over 1&nbsp;% of the entries Honey has collected on Benni. We have focused on non-product related page views. But in addition, Honey also knows every product that Benni looked at while he had installed the extension. Further, they could use the data they have for even more conclusions and profiles. For example, they could infer sleep cycles from the timestamps or build interest profiles based on the sites that were visited.
```

Comment:
"focussed" is also fine: https://en.wiktionary.org/wiki/focussed

Reply:
Focused is the most common and widely accepted spelling of this adjective. Focussed is an old spelling that is still sometimes accepted but much less common. 

The confusion about the spelling of focused comes because you need to double the consonant when you have a single letter stressed vowel preceding it.

You do not need to double the consonant in focused because the stress is on the first “o”.


Sure, so would you like I leave it with double "s"?

Reply:
> Sure, so would you like I leave it with double "s"?

Yes, please do.